application/plugin_base/item_list.py

```diff
@@ -1647,6 +1665,19 @@ def destroy_providers(self):
 		for path, provider in self._providers.items():
 			provider.release_archive_handle()
 
+	def create_intermediate_archive_provider(self, path=None):
```

Comment:
Okay, I'll review the code later today. Sorry for late replies, I got ill in the mean time.